jni/Application.mk

```diff
@@ -1,3 +1,3 @@
 APP_PLATFORM := android-16
-APP_ABI := armeabi-v7a arm64-v8a x86 x86_64
+APP_ABI := arm64-v8a
```

Comment:
it seems this sneaked in while you was testing

Reply:
Ooops, thanks for watching out!